app/renderer/tray/index.js

```diff
@@ -8,6 +8,7 @@ import Routes from './routes';
 
 
 const renderApp = routes => {
+  console.log('RenderApp');
```

Comment:
Stray debug log got left in 😁

Reply:
This was the only thing I noticed out of place, otherwise everything looks good!

Reply:
haha removed it 😄